src/main/java/racingcar/model/CarList.java

```diff
@@ -0,0 +1,54 @@
+package racingcar.model;
+
+import racingcar.util.IntegerConst;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+public class CarList {
+    private final ArrayList<Car> carList;
+
+    public CarList(String[] names) {
+        this.carList = new ArrayList<>();
+        for (String eachName : names) {
+            this.carList.add(new Car(eachName));
+        }
+    }
+
+    public ArrayList<String> getNameList() {
+        ArrayList<String> nameList = new ArrayList<>();
+        for (Car car : this.carList) {
+            nameList.add(car.getName());
+        }
+        return nameList;
```

Comment:
`ArrayList`에는 `Collection`을 인자로 받는 생성자도 있습니다.

```suggestion
        return new ArrayList(nameList);
```

Reply:
제안해주신 변경사항에 nameList는 혹시 Car의 이름들을 담은 collection인지 궁금합니다!

Reply:
앗, 이름을 따로 빼서 리스트를 만드는군요. 제가 착각했네요 😅